kpi-tracker/README.md

```diff
@@ -2,16 +2,47 @@
 
 This folder contains the source code for the KPI-tracker service used to collect relevant metrics from the chain.
```

Comment:
It would be useful to list here which metrics are tracked.

Reply:
Yes. Will add this.

Reply:
I added a section pertaining to the database design.